test/Data/OpenApi/Schema/GeneratorSpec.hs

```diff
@@ -69,22 +69,22 @@ spec = do
     prop "TL.Text" $ shouldValidate (Proxy :: Proxy TL.Text)
     prop "[String]" $ shouldValidate (Proxy :: Proxy [String])
     -- prop "(Maybe [Int])" $ shouldValidate (Proxy :: Proxy (Maybe [Int]))
-    prop "(IntMap String)" $ shouldValidate (Proxy :: Proxy (IntMap String))
+    -- prop "(IntMap String)" $ shouldValidate (Proxy :: Proxy (IntMap String))
     prop "(Set Bool)" $ shouldValidate (Proxy :: Proxy (Set Bool))
     prop "(NonEmpty Bool)" $ shouldValidate (Proxy :: Proxy (NonEmpty Bool))
     prop "(HashSet Bool)" $ shouldValidate (Proxy :: Proxy (HashSet Bool))
     prop "(Either Int String)" $ shouldValidate (Proxy :: Proxy (Either Int String))
-    prop "(Int, String)" $ shouldValidate (Proxy :: Proxy (Int, String))
+    -- prop "(Int, String)" $ shouldValidate (Proxy :: Proxy (Int, String))
```

Comment:
are these commented tests expected ? didn't want to update the template schema ?

Reply:
Those tests are checking that for any JSON value generated matching the schema generated by the `ToSchema` instance of `(Int, String)`, we are able to parse it to `(Int, String)`.

An example might make it clearer, we try to check whether we can parse `[1, "hello"]` to a `(Int, String)` which we can.
However, since the only constraints we have is having two elements of type `Int` or `String`, we also try `[1, 1]`, `["hello", "bye"]` and `["hello", 1]` which we cannot parse to `(Int, String)`.

Reply:
makes sense